### ISSUE TITLE
Import directly into journal?

### ISSUE THREAD

**Issue by regexident** [omnivore-app/logseq-omnivore#124]
I'd like to import entries from Omnivore directly into the Journal pages corresponding to ```{{{dateSaved}}}``` (as my use of Omnivore is mainly as a trampoline for Logseq).

I tried setting `pageName` to ```{{{dateSaved}}}```, but the plugin now doesn't seem to import anything anymore, instead.

And I'm getting conflicts (containing entries from Omnivore) for `pages/.md`.

**Comment by jacksonh** [comment]
Hi @regexident can you try `{{{date}}}` ?

**Comment by regexident** [comment]
Yes, that's what I initially tried (since it was suggested by the settings label). Stumbling upon [this](https://github.com/omnivore-app/logseq-omnivore/blob/7b2ec76bf8a37ff04a8a7aafd16c338289fe2817/src/settings/template.ts#L68) I figured the label might just be outdated and tried `dateSaved`.

I changed the `syncAt` to an earlier date (~2020) to force a re-import, but I'm not seeing any imports at all. Neither in the journal, nor in pages corresponding to the date, nor anywhere else.

**Comment by jacksonh** [comment]
@regexident thanks. Do you mind opening developer tools in Logseq, clicking on the console tab, and seeing if there is any info? It looks like you are right it should be `{{{dateSaved}}}` I think.

**Comment by regexident** [comment]
The only errors the console revealed where a whole bunch of these:
<img width="537" alt="image" src="https://github.com/omnivore-app/logseq-omnivore/assets/138017/7743a379-5485-4c2b-b46b-2874fb9fa291">
(Which also seem to have correlated with flickering of the UI.)

The rest consists of logs like this one:
![image](https://github.com/omnivore-app/logseq-omnivore/assets/138017/00a57e9c-9294-4ba7-97be-c12f8e9aee43)


Here are the complete logs (after clearing it before triggering an import): [console.txt](https://github.com/omnivore-app/logseq-omnivore/files/12239624/console.txt)

**Comment by jacksonh** [comment]
Thanks, one more thing, can you open the command pallette (CMD+SHIFT+p) and run `Resync all Omnivore Articles` ?

**Comment by regexident** [comment]
Cleared the console, and executed `Resync all Omnivore Articles`, resulting in the following logs: [console.txt](https://github.com/omnivore-app/logseq-omnivore/files/12239889/console.txt)

No errors are logged this time.

Searching for the most recent Omnivore entry (or any other, for that matter) in Logseq still finds nothing.

**Comment by regexident** [comment]
I'm running the following versions:

Logseq v0.9.13 (66)
Omnivore v1.13.3

**Comment by regexident** [comment]
@sywhb sure!

```json
{
  "generalSettings": "",
  "apiKey": "<SNIP>",
  "filter": "import all my articles",
  "customQuery": "",
  "frequency": "10",
  "graph": "Personal",
  "syncAt": "2023-08-03T11:06:36",
  "highlightOrder": "the location of highlights in the article",
  "isSinglePage": false,
  "pageName": "{{{dateSaved}}}",
  "articleTemplate": "[{{{title}}}]({{{originalUrl}}})",
  "highlightTemplate": "> {{{text}}} [⤴️]({{{highlightUrl}}}) {{#labels}}#[[{{{name}}}]] {{/labels}}",
  "advancedSettings": "",
  "endpoint": "https://api-prod.omnivore.app/api/graphql",
  "disabled": false,
  "version": "v1.13.3",
  "syncJobId": 10,
  "loading": false
}
```

**Comment by sywhb** [comment]
@regexident this looks all correct to me except that the `pageName` should be `{{{date}}}`. 

I know it is a bit confusing but I used a different variable name here.

**Comment by regexident** [comment]
Hm, not sure why but with `{{{date}}}` (instead of `{{{date}}}`) it now finally did something. I had tried both in the past.

But now it created hundreds of individual pages named after the dates. I'd like the entries to instead get (appended or prepended) to journal entries corresponding to the given dates, not separate pages. 😕

(I'd effectively like "save entry to omnivore" to be a shortcut for "manually add entry to today's Logseq journal". Especially for saving stuff to Logseq from a mobile device.)

Would a feature like this be in scope for this plugin?

**Comment by sywhb** [comment]
@regexident It's kind of weird because the synced articles are entries in the journal in my Logseq after I sync with Omnivore.

<img width="901" alt="Screenshot 2023-08-04 at 10 42 23 AM" src="https://github.com/omnivore-app/logseq-omnivore/assets/12963965/da8083e4-3fb1-4a27-9720-44ac4d7ede7d">

Would you mind checking if `isSinglePage` is not checked?
<img width="616" alt="Screenshot 2023-08-04 at 12 28 05 PM" src="https://github.com/omnivore-app/logseq-omnivore/assets/12963965/00209c3c-13cf-41fd-981d-d3f9bcf7c812">

**Comment by sywhb** [comment]
@regexident Never mind, I noticed you have `isSinglePage: false` in the settings.

May I know the date format in your journal and the synced page name?

**Comment by regexident** [comment]
Update: I just went ahead and unified journal names to `2023_01_23.md` and re-indexed the graph.

It looks like the plugin works as intended now. To verify I added an entry to Omnivore, re-fetched via the plugin in Logseq and the entries and it appended the following to today's journal:

```
- ## 🔖 Articles
	- <SNIP>
```

So I guess the issue can be closed (I'll leave that to you). Thank you both so much for your help @jacksonh & @sywhb!
